### PR TITLE
#22 add experts preview component

### PR DESCRIPTION
**Pull request by tania833**
develop

## GitHub Board

**Issue link**

[#22  ]( )

**Story link**

[#22 ]( )

## Code reviewers
- [ ] @github_username

## Summary of issue

- [ ]  Summary of issue
- [ ]  Summary of issue

## Summary of change

- [ ]  Summary of change
- [ ]  Summary of change

### CODE REVIEW

File: src/lib/components/ExpertBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import cards from '../constants/mockDataCards';
+
+interface IExpertProps{
+  infoCards: typeof cards;
+}
```
Comment (SergeOlabin) [inline src/lib/components/ExpertBlock.tsx:6]:
```suggestion
interface IExpertProps{
  expert: IExpert, // imported from lib/types.ts
}
```

File: src/lib/components/ExpertBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          <div className="card-image waves-effect waves-block waves-light">
+            <img className="activator" src={card.photo} alt="doctor" />
+          </div>
+        </div>
+      </div>
+    );
+  })}</div>
+    </div>
+  );
+};
```
Comment (SergeOlabin) [inline src/lib/components/ExpertBlock.tsx:26]: pls don't do that.

you can create another variable for 
```
{infoCards.map((card) => {
    return (
      <div className="col m3" key={card.phone}>
        <div className="card small">
          <div className="card-image waves-effect waves-block waves-light">
            <img className="activator" src={card.photo} alt="doctor" />
          </div>
        </div>
      </div>
    );
  })}
```
and use it

File: src/lib/constants/mockDataCards.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { IExpert, ExpertStatus } from '../types';
```
Comment (SergeOlabin) [inline src/lib/constants/mockDataCards.tsx:1]: move to modules/experts,

the content is nice!

File: src/lib/components/ExpertBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+interface IExpertProps{
+  infoCards: typeof cards;
+}
+
+export const ExpertBlock:React.FC<IExpertProps> = (props)=> {
+  const {infoCards} = props;
+  return (
+    <div>
+      <h4>Experts</h4>
```
Comment (SergeOlabin) [inline src/lib/components/ExpertBlock.tsx:12]: remove header,
this component should be related to the only 1 expert card

**Review by SergeOlabin — CHANGES_REQUESTED** [review]

File: src/lib/components/ExpertBlock.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  expert: IExpert[];
+}
+
+export const ExpertBlock: React.FC<IExpertProps> = (props) => {
+  const { expert } = props;
+  const Cards = expert.map((card) => {
+    return <img src={card.photo} alt="doctor" key={card.phone} />;
+  });
+  return <div>{Cards}</div>;
+};
```
Comment (SergeOlabin) [inline src/lib/components/ExpertBlock.tsx:14]: expert block should be per 1 expert

File: src/lib/components/ExpertBlock.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { IExpert } from '../types';
+
+interface IExpertProps {
+  expert: IExpert[];
```
Comment (SergeOlabin) [inline src/lib/components/ExpertBlock.tsx:5]: expert: IExpert

**Review by SergeOlabin — CHANGES_REQUESTED** [review]

File: src/lib/components/ExpertBlock.tsx (hunk 6 above)
Comment (tania833) [reply src/lib/components/ExpertBlock.tsx]: done

File: src/lib/components/ExpertBlock.tsx (hunk 5 above)
Comment (tania833) [reply src/lib/components/ExpertBlock.tsx]: done

File: src/lib/components/Experts.tsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ExpertBlock } from './ExpertBlock';
+import cards from '../../modules/experts/mockDataCards';
+
+export const Experts: React.FC = () => {
+  const allExperts = cards.map((card) => {
```
Comment (SergeOlabin) [inline src/lib/components/Experts.tsx:6]: Cool!

**Review by SergeOlabin — APPROVED** [review]

**Pull request merged**